source/rm-openapi-latest.json

```diff
@@ -375,6 +375,316 @@
           "Project"
         ]
       }
+    },
+    "/project/{projectId}/codegen/generate": {
```

Comment:
It looks OK, especially if it's been generated by the `npx openapi-format...` command that Alex has noted. Would it be possible to pull the OpenAPI Json files from our S3 bucket or is this flow better where you can generate it out of the repo?

Reply:
We could pull from the s3 bucket, but using the command allows us to publish docs before a release, which we discussed on previous call is better than being later. If we wait, we need to wait until the new release is actually published. This way allows us to get the docs up 2-3 hours before the release in a single PR. @alwaysbemark